core/src/main/java/io/ultrabrew/metrics/data/BasicCounterAggregator.java

```diff
@@ -37,35 +37,35 @@ public BasicCounterAggregator(final Counter counter) {
   /**
    * Create a monoid for common aggregation functions for a Counter.
    *
-   * @param metricId identifier of the metric associated with this aggregator
+   * @param metricName identifier of the metric associated with this aggregator
```

Comment:
Should the Javadoc also be updated to say "name" instead of "identifier" if we're going for a terminology change?

Reply:
Done.

Reply:
Why do you want to change from a specific term describing the intent and purpose of the variable to a name that is NOT precise and can be misunderstood as a label type instead of what it truly is: an identifier. Also this breaks consistency and backwards compatibility in the library. Do not make this change

Reply:
It's not an internal id but the actual name of the metric provided by the user. So, I was trying to be consistent with the semantics. But, I agree that it will break consistency. I will revert my last commit.

Reply:
There is no such things as internal id, there is public id of the metric that the developer defines.